demisto_sdk/commands/validate/validators/GR_validators/GR100_uses_items_not_in_market_place.py

```diff
@@ -0,0 +1,115 @@
+from demisto_sdk.commands.content_graph.objects.job import Job
+from demisto_sdk.commands.content_graph.objects.layout import Layout
+from demisto_sdk.commands.content_graph.objects.layout_rule import LayoutRule
+from demisto_sdk.commands.content_graph.objects.mapper import Mapper
+from demisto_sdk.commands.content_graph.objects.modeling_rule import ModelingRule
+from demisto_sdk.commands.content_graph.objects.pack import Pack
+from demisto_sdk.commands.content_graph.objects.parsing_rule import ParsingRule
+from demisto_sdk.commands.content_graph.objects.playbook import Playbook
+from demisto_sdk.commands.content_graph.objects.report import Report
+from demisto_sdk.commands.content_graph.objects.script import Script
+from demisto_sdk.commands.content_graph.objects.test_playbook import TestPlaybook
+from demisto_sdk.commands.content_graph.objects.trigger import Trigger
+from demisto_sdk.commands.content_graph.objects.widget import Widget
+from demisto_sdk.commands.content_graph.objects.wizard import Wizard
+from demisto_sdk.commands.content_graph.objects.xsiam_dashboard import XSIAMDashboard
+from demisto_sdk.commands.content_graph.objects.xsiam_report import XSIAMReport
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Union[
+    Integration,
+    Script,
+    Playbook,
+    Pack,
+    Dashboard,
+    Classifier,
+    Job,
+    Layout,
+    Mapper,
+    Wizard,
+    CorrelationRule,
+    IncidentField,
+    IncidentType,
+    IndicatorField,
+    IndicatorType,
+    LayoutRule,
+    Layout,
+    ModelingRule,
+    ParsingRule,
+    Report,
+    TestPlaybook,
+    Trigger,
+    Widget,
+    GenericDefinition,
+    GenericField,
+    GenericModule,
+    GenericType,
+    XSIAMDashboard,
+    XSIAMReport,
+]
+
+
+class MarketplacesFieldValidator(BaseValidator[ContentTypes]):
+    error_code = "GR100"
+    description = (
+        "Content item attempts to use other content items which are not supported in all of the "
+        "marketplaces of the content item."
+    )
+    rationale = "Content graph proper construction."
+    error_message = ""
+    related_field = "marketplaces"
+    is_auto_fixable = False
+    expected_git_statuses = [
+        GitStatuses.RENAMED,
+        GitStatuses.ADDED,
+        GitStatuses.MODIFIED,
+    ]
+
+    def is_valid(self, content_items: Iterable[ContentTypes]) -> List[ValidationResult]:
+        validation_results = []
+
+        for content_item in self.graph.find_uses_paths_with_invalid_marketplaces(
+            [item.pack_id for item in content_items]
+        ):
+
+            used_content_items = [
+                item.content_item_to.object_id
+                for item in content_item.relationships_data.get(RelationshipType.USES)
+            ]
+
+            error_message, _ = Errors.uses_items_not_in_marketplaces(
+                content_item.name, content_item.marketplaces, used_content_items
+            )
```

Comment:
We don't want to use the old errors.
you should copy/alter the message and put it under error_message.
If you wish to send any variables use format (see other validations as a reference).

Reply:
adapted the error message construction.